sebex/analysis/state.py

```diff
@@ -1,11 +1,13 @@
+from sebex.analysis.version import Version
+from sebex.config.manifest import ProjectHandle
 from typing import Tuple
 
 from sebex.analysis.database import AnalysisDatabase
 from sebex.analysis.graph import DependentsGraph
 from sebex.config.profile import current_project_handles
 
 
-def analyze() -> Tuple[AnalysisDatabase, DependentsGraph]:
-    database = AnalysisDatabase.collect(current_project_handles())
+def analyze(bumps: dict[ProjectHandle, Version] = None) -> Tuple[AnalysisDatabase, DependentsGraph]:
```

Comment:
Either upgrade Python to 3.9 where lowercase dict deprecated uppercase typing.Dict one, or:

```suggestion
from typing import Dict

def analyze(bumps: Dict[ProjectHandle, Version] = None) -> Tuple[AnalysisDatabase, DependentsGraph]:
```